Mathlib/Algebra/Ring/Defs.lean

```diff
@@ -109,7 +112,8 @@ that `Semiring -> NonAssocSemiring` is tried before `NonAssocRing -> NonAssocSem
 TODO: clean this once lean4#2115 is fixed
 -/
 
-/-- A not-necessarily-unital, not-necessarily-associative semiring. -/
+/-- A not-necessarily-unital, not-necessarily-associative semiring. See `CommutatoRing` and the documentation thereof
+in you need to out a `NonUnitalNonAssocSemiring` instance on a Lie ring/algebra. -/
```

Comment:
```suggestion
/-- A not-necessarily-unital, not-necessarily-associative semiring. See `CommutatoRing` and the documentation thereof in case you need a `NonUnitalNonAssocSemiring` instance on a Lie ring/algebra. -/
```

Reply:
Should `CommutatoRing` be `CommutatorRing`?

Reply:
Thanks! Fixed before sending to CI 😄